keyboardjs/keyboardjs.d.ts

```diff
@@ -1,50 +1,135 @@
-// Type definitions for KeyboardJS
+// Type definitions for KeyboardJS v2.2.0
 // Project: https://github.com/RobertWHurst/KeyboardJS
-// Definitions by: Vincent Bortone <https://github.com/vbortone/>

```

Comment:
Since this d.ts is a full replacement of the old one due to the complete API change I thought it wouldn't be necessary. Done anyway :)